src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -13,27 +13,23 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-import java.io.IOException;
+
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.datastore.v1.PropertyFilter;
-import com.google.sps.servlets.User;
 import com.google.gson.Gson;
-import java.util.HashMap;
-import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.List;
-
+import org.apache.commons.io.IOUtils;
+import com.google.sps.servlets.User;
 
-FirebaseApp.initializeApp();
 /** Servlet that holds the users on this WebApp */
```

Comment:
this comment is similar to the one on UsersServlet.java, line 34. Do they both do the same thing?

Reply:
The ioutils? Yes they do

Reply:
i was referring to the comment /** Servlet that holds the users on this WebApp */